extra-libs/libftdi/autobuild/beyond

```diff
@@ -1,3 +1,4 @@
 abinfo "Moving misplaced files in /usr/lib64 ..."
+mkdir -pv "$PKGDIR"/usr/lib/
 cp -rv "$PKGDIR"/usr/lib64/* "$PKGDIR"/usr/lib/
```

Comment:
You might want to add an if branch here.

Reply:
what if branch? check whether /usr/lib64 exists?

Reply:
Yes